### ISSUE TITLE
Update README for the python interpreter

### ISSUE THREAD

**Issue by mstfdkmn** [irods/irods_rule_engine_plugin_python#188]
I guess [this](https://github.com/irods/irods_rule_engine_plugin_python?tab=readme-ov-file#python-version) section needs to be updated since Python3 Compliance is ensured with the iRODS 4.3.0.

**Comment by trel** [comment]
That is correct - it's using the default Python interpreter on the system that compiles the plugin.

Thanks.
